helpers/setup_next_run.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python
 
+### /glade/u/apps/opt/conda/envs/npl/bin/python ??
```

Comment:
`/usr/bin/env python` should find `/glade/u/apps/opt/conda/envs/npl/bin/python` if your environment is set up to use it(?)  We shouldn't hard code that path, because everyone will likely use a slightly different python version / path

Reply:
ok